.travis.yml

```diff
@@ -1,6 +1,12 @@
 rvm: 2.2.3
 sudo: false
 cache: bundler
+addons:
+  apt:
+    sources:
+    - elasticsearch-2.1
+    packages:
+    - elasticsearch
```

Comment:
The reason why I moved the Travis config to Trusty was because the elasticsearch-2.1 source doesn't actually work. They are also not adding any more whitelisted sources, so the only way to test against ES 2.x is to either use Elastic.co's package repository (which only provides the latest 2.x package), or install a specific package from their archive download.